README.md

```diff
@@ -8,40 +8,25 @@
 
 <img src="docs/diagrams/logo.png" width="200">
 
-## Introduction to the Postgres Operator
+The Postgres operator manages PostgreSQL clusters on Kubernetes. Configuration
+happens only via manifests. Therefore, it integrates easily in automated CI/CD
+pipelines with no access to Kubernetes directly.
 
-The Postgres [operator](https://coreos.com/blog/introducing-operators.html)
```

Comment:
Where did this go?

Reply:
"Moved" to index.md (concepts). It already contained the same description as the README except for the 4th bullet point.